polymer.json

```diff
@@ -2,9 +2,10 @@
   "entrypoint": "index.html",
   "shell": "src/start-polymer3.js",
   "fragments": [
+    "src/lazy-element.js"
```

Comment:
The Polymer CLI does not recognize the dynamic imports?

Reply:
Correct, it does not. This project won't build, and will only work on the latest versions of Chrome and Safari.